src/XenCert/StorageHandler.py

```diff
@@ -2165,8 +2165,8 @@ def Create(self, device_config = {}):
     def GetPathStatus(self, device_config):
         # Query DM-multipath status, reporting a) Path checker b) Path Priority handler c) Number of paths d) distribution of active vs passive paths
         try:
-            self.mapIPToHost = StorageHandlerUtil._init_adapters()      
-            XenCertPrint("The IP to host id map is: %s" % self.mapIPToHost) 
+            self.mapHostToIP = StorageHandlerUtil._init_adapters()      
+            XenCertPrint("The IP to host id map is: %s" % self.mapHostToIP) 
```

Comment:
The Printed message mis-matched the parameter name.
The parameter name says "Map Host to IP", the printed message says "The IP to Host ID Map".
which is confusing indeed. can we make it correct?